core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/DatalakeV4Controller.java

```diff
@@ -11,14 +11,17 @@
 import com.sequenceiq.authorization.annotation.InternalOnly;
 import com.sequenceiq.cloudbreak.api.endpoint.v4.common.StackType;
 import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.DatalakeV4Endpoint;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.request.DiskUpdateRequest;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.request.SdxDiskUpdateRequest;
 import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.response.StackViewV4Responses;
 import com.sequenceiq.cloudbreak.auth.security.internal.TenantAwareParam;
+import com.sequenceiq.cloudbreak.cloud.model.VolumeSetAttributes;
 import com.sequenceiq.cloudbreak.common.exception.BadRequestException;
+import com.sequenceiq.cloudbreak.service.datalake.DiskUpdateService;
 import com.sequenceiq.cloudbreak.service.workspace.WorkspaceService;
 import com.sequenceiq.distrox.v1.distrox.StackOperations;
 
 @Controller
-@InternalOnly
```

Comment:
do not remove this, this is an internal API

Reply:
@horadla23 i have removed this from the class level to the method that was using it. so the list api which was internal only - will still remain internal. I dont want my api to be internal only

Reply:
this is an old API from pre Cloudera era, now all public facing datalake API must be in datalake service, all datalake related API in core **must be internal**, we made the whole class internal for a reason